src/api/config/boot.rb

```diff
@@ -1,3 +1,5 @@
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
+if !File.exist?(File.expand_path('../Gemfile.in', __dir__))
```

Comment:
```suggestion
unless File.exist?(File.expand_path('../Gemfile.in', __dir__))
```

Reply:
"Unless" if more difficult to parse then "if not" for non-English speakers.
Doing this anyway, just because your CI doesn't requires it.